common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -114,10 +146,89 @@ static String getOsPrincipalClassName() {
 
     @Override
     public AppConfigurationEntry[] getAppConfigurationEntry(String appName) {
-      // TODO: switch branch based on appName. (Simple, Kerberos, etc)
-      return SIMPLE;
+      if (appName.equalsIgnoreCase(AuthenticationFactory.AuthTypes.SIMPLE.getAuthName())) {
+        return SIMPLE;
+      } else if (appName.equalsIgnoreCase(AuthenticationFactory.AuthTypes.KERBEROS.getAuthName())) {
+        // TODO: return KERBEROS;
+        throw new UnsupportedOperationException("Kerberos is not supported currently.");
+      }
+      return null;
+    }
+  }
+
+  /**
+   * Set the static configuration and initialize based on it
+   * @param conf the configuration
+   */
+  public static void setTachyonConf(TachyonConf conf) {
```

Comment:
I guess there might be cases requiring setting the conf dynamically, but still not catch it. Maybe there is not. So this is only for test purpose now.

Is there an example in Tachyon code about the alternative solution to work around? I could take a look and make updates following that way.